logos/src/logos/capacity/capacity_planner.py

```
@@ -1128,6 +1128,21 @@ def _compute_idle_actions(
         now = time.time()
         actions = []
 
+        # Never sleep the only usable lane on a worker — it would leave
+        # the worker with zero serving capacity.
+        active_lanes = [
+            l for l in lanes
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)